### ISSUE TITLE
None of the actionable status bar items have aria-labels

### ISSUE THREAD

**Issue by ramya-rao-a** [Microsoft/vscode#37343]
Most status bar items that are actionable are implemented using `<a>` tag
They have the `title` attribute set that displays the tooltip.

But for people using screenreaders, this is bad design.

The status bar items should be accompanied appropriate by aria-labels that describe what the status as well as the action behind the status bar item is

**Comment by ramya-rao-a** [comment]
On a second thought, while tabbing around, the items in the status bar are never reached either

**Comment by bpasero** [comment]
@ramya-rao-a this is actually "as designed" because we did not want to frustrate users that want to tab through the UI that end up in countless status bar entries. The rationale is that any command down there is also accessible from the F1 list.

**Comment by ramya-rao-a** [comment]
good point
